### PR TITLE
completes task 3 - path api/topics get requests and error handling

### PR DESCRIPTION
**Pull request by Wyclif03**
Task complete with for 3. GET created for path api/topics and first round of error handling

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+})
+test('it should return the keys slug and description for each item in the array', () =>{
+    return request(app)
+    .get('/api/topics')
+    .expect(200)
+    .then(({body}) => {
+        const {topics} = body
+        topics.forEach((topic) => {
+            expect(Object.keys(topic)).toEqual(["slug", "description"]);
+})
```
Comment (RichardNeat) [inline __tests__/api.test.js:57]: Nice testing here Paul the only thing I'd say we aren't confirming is the data types of the values coming through. Have a look at the way we've done it over previous sprints by using a forEach, toMatchObject and expect.any to match the exact properties and value types in the response.

We also need to be checking the length of the response array (we know how long the response should be by looking at the test data that's being seeded beforeEach test).

Also you're fine to combine these expect assertions into one test block which may make it easier to write out :) We can generally deal with all happy path assertions in one block.

File: db/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```
Comment (RichardNeat) [inline db/app.js]: We don't need this yet as we aren't handling and input bodies so please remove :)

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice one Paul, just a couple of requested changes please. Can you make those amendments and then pop me a private message with a link to them. Cheers

**Comment by Wyclif03** [comment]
Updated with new request

**Pull request closed without merging**